client/src/components/Form/Form.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
These are the same changes as in this PR https://github.com/pesto-students/batch-8-CODERHOOD/pull/62